src/pro/javacard/gp/GPTool.java

```diff
@@ -115,7 +115,11 @@
 	private final static String OPT_VERSION = "version";
 	private final static String OPT_VIRGIN = "virgin";
 	private final static String OPT_VISA2 = "visa2";
-
+	private final static String OPT_ACR_LIST = "acr-list";
+	private final static String OPT_ACR_ADD = "acr-add";
+	private final static String OPT_ACR_DELETE = "acr-delete";
+	private final static String OPT_ACR_RULE = "rule";
```

Comment:
Maybe prefix the rest of the ACR(?) related options also with -acr, -acr-rule and -acr-hash ?